conferences/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^(?P<zosia_id>\d+)/register$', views.register, name='user_zosia_register'),
```

Comment:
tak jest chyba bardziej elastycznie, zawsze mozna miec te kontrolki wylaczone jesli to jest nieaktywna zosia